omero/developers/whatsnew.txt

```diff
@@ -7,7 +7,9 @@ What's new for OMERO 5.1
   (currently slated to be 5.2). Most notably for development purposes, we are
   deprecating support for Java 1.6 and Python 2.6.
 - :doc:`Model/Units` introduces the new Units-types which are available
-  for use from all model objects.
+  for use from :doc:`Model Objects <Model/EveryObject>` (note that this
```

Comment:
One example would be that the first query under
https://www.openmicroscopy.org/site/support/omero5.1-staging/developers/Model/Units.html#querying-units
was previously:

```
   select planeInfo.exposureTime from PlaneInfo planeInfo
```

which would return a floating point number. Now to get the floating
point number, one must add ".value" **but** that's not sufficient for
the purposes of comparison since one needs the enum to go along with
it.